tests/Functional/Controller/SmokeFunctionalControllerTest.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+namespace App\Tests\Functional\Controller;
+
+use RuntimeException;
+use Symfony\Bundle\FrameworkBundle\Test\WebTestCase;
+use Throwable;
+
+final class SmokeFunctionalControllerTest extends WebTestCase
+{
+    const ANONYMOUS = 'anom.';
+
+    /**
+     * @return void
+     */
+    public function testSmokeTest()
+    {
+        foreach ($this->getUrlsToTest() as $account => $paths) {
+            $client = static::createClient();
+
+            if (self::ANONYMOUS !== $account) {
+                //TODO Login
+            }
+            foreach ($paths as $url) {
+                $method = 'GET';
+                $data = [];
+                if (is_array($url)) {
+                    $method = isset($url['method']) ? $url['method'] : $method;
+                    $data = isset($url['data']) ? $url['data'] : $data;
+                    $url = isset($url['url']) ? $url['url'] : null;
+                }
+
+                if (null === $url) {
+                    throw new RuntimeException('Url cannot be null.');
+                }
+
+                try {
```

Comment:
Este `try` esta de más, usando los metodos `$this->assert*(...)` si no se cumple lo que esperamos los test truenan

Reply:
removi los try